campaigns/sync.py

```
@@ -0,0 +1,649 @@
+            "account_id": account_id,
+            "method_name": method_name,
+            "args_base64": args_base64,
+            "finality": "optimistic",
+        },
+    }
+
+    response = requests.post(rpc_url, json=payload, timeout=15)
+    result = response.json()
+
+    if "error" in result:
+        raise Exception(f"RPC error: {result['error']}")
```

Ruff (0.15.1)

[warning] Create your own exception

(TRY002)